xbmc/FileItem.cpp

```diff
@@ -490,8 +476,7 @@ void CFileItem::Reset()
   m_epgInfoTag.reset();
   delete m_pvrChannelInfoTag;
   m_pvrChannelInfoTag=NULL;
-  delete m_pvrRecordingInfoTag;
-  m_pvrRecordingInfoTag=NULL;
+  m_pvrRecordingInfoTag.reset();
```

Comment:
So you're just questioning the sense of this PR as a whole? Wasn't it the intention no more to use copies and to always act upon the one end only instance? Confused... :-/